Client/Frontend/Settings/SettingsViewController.swift

```diff
@@ -11,6 +11,8 @@ import LocalAuthentication
 import SwiftyJSON
 import Data
 import WebKit
+import BraveRewards
+import class BraveRewardsUI.QASettingsViewController
```

Comment:
Have to specify the class import because blanket import causes ambiguous symbols for `Strings.Settings` (tracked: https://github.com/brave/brave-rewards-ios/issues/214)

Reply:
Updated in brave/brave-rewards-ios#215